23.0rc5/sipsorcery/noncodesigned.SHA256SUMS.asc

```diff
@@ -1,17 +1,16 @@
 -----BEGIN PGP SIGNATURE-----
-
```

Comment:
I think something went wrong here? You also have a merge commit for some reason.

Reply:
I get a merge conflict when I add code signed, after non-code signed, attestations.

Reply:
I'll try this again.